design/components/cards.stories.tsx

```diff
@@ -16,20 +19,34 @@ const Template: Story<CardsProps> = (args): React.ReactElement => (
 
 export const CaseWithImage = Template.bind({});
 CaseWithImage.args = {
-  service: SERVICE,
-  image: { url: duurzaamheid },
-  client: 'cyclo',
-  title: 'Eenvoudig fietsparkeren',
   href: '/',
-  target: '_blank',
+  children: [
+    <img alt="decorative" src={duurzaamheid} />,
+    <ContentContainer>
+      <main>
+        <Tag>Cyclo</Tag>
+        <Title as="h3" size={3}>
+          Eenvoudig fietsparkeren
+        </Title>
+        <RichText>{RICH_TEXT}</RichText>
+      </main>
+    </ContentContainer>,
+  ],
```

Comment:
I don't think a seperately exported `ContentContainer` and usage of `main` should be necessary.

I expect the story to be more something like this.

```ts
export const Default = Template.bind({});
Default = {
  image: duurzaamheid,
  children: [
      <Tag>Theme</Tag>,
      <Title size={3}><a href="/">Eenvoudig fietsparkeren</a></Title>,
      <RichText>{RICH_TEXT}</RichText>,
  ],
};
```

Reply:
the car is ment to be clickable you made only the title as link?

Reply:
i did remove the main and ContentContainer from the children but i had to add extra divs for styling in card.tsx

Reply:
Done as you recommended